pandas/tests/indexes/common.py

```diff
@@ -344,18 +345,46 @@ def test_copy_and_deepcopy(self, indices):
 
     def test_duplicates(self, indices):
         if type(indices) is not self._holder:
-            return
+            pytest.skip('Can only check if we have the correct type')
         if not len(indices) or isinstance(indices, MultiIndex):
-            return
+            # MultiIndex tested separately in:
+            # tests/indexes/multi/test_unique_and_duplicates
+            pytest.skip('Skip check for empty Index and MultiIndex')
+
         idx = self._holder([indices[0]] * 5)
         assert not idx.is_unique
         assert idx.has_duplicates
 
+    @pytest.mark.parametrize('keep', ['first', 'last', False])
```

Comment:
is this a duplicate these (pun intended) or are not testing indices duplicated currently?

Reply:
@jreback `.duplicated` is hardly ever tested directly, only indirectly for stuff like `.drop_duplicates`. Regardless of the changes to `.duplicated` in #21645, I think duplicated should be tested separately.

Reply:
so these are not relevant?
```
pandas/tests/indexes/common.py:    def test_duplicates(self, indices):
pandas/tests/indexes/test_category.py:    def test_duplicates(self):
pandas/tests/indexes/test_range.py:    def test_duplicates(self):
```

Reply:
@jreback No, `test_duplicates` (at least for `pandas/tests/indexes/common.py`, which is what this PR is about) tests `.is_unique` and `.has_duplicates`, but not the `.duplicated`-method itself.

Reply:
can you point to the coverage that shows this is NOT tested?

Reply:
@jreback I never said that it is not tested, just that it is only tested *implicitly*. Any call to `.drop_duplicates` will invoke `duplicated`, and so obviously the coverage works out.